welkin/models/cdt.py

```diff
@@ -2,39 +2,45 @@
 
 from welkin.models.base import Collection, Resource
 from welkin.pagination import PageNumberIterator
+from welkin.util import model_id
 
 
 class CDT(Resource):
-    def create(self):
+    @model_id("Patient")
+    def create(self, patient_id: str):
```

Comment:
Adding a positional argument to a method would usually trigger a higher-level version bump, this change is technically idempotent and fully backwards-compatible since `model_id` will set the argument.